doc/dune-files.rst

```diff
@@ -327,6 +327,12 @@ OCaml/Reason modules only consume modules from the directory where the
 stanza appear. In order to declare a multi-directory library, you need
 to use the :ref:`include_subdirs` stanza.
 
+Note also that a nested library in a project is not automatically namespaced
```

Comment:
The term "nested library" does not exist in the documentation, so I'm not sure what you're referring to?

Reply:
I'm referring to libraries in subdirectories of libraries. Is there are more standard terminology used for that? If so I can easily change to that.